Unity-Technologies/ScriptableRenderPipeline
# Fix for Blend Node NaN Output

### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
Adding padding to the input values of the Blend node to prevent NaN outputs.
---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.
Adding padding to the input values of the Blend node to prevent NaN outputs.
---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
Green
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?automation-tools_branch=master&ScriptableRenderLoop_branch=sg%2Fblend-node-fix&unity_branch=trunk

**Manual Tests**: What did you do?
- [x] ~Built a player~ Editor-only code changes
- [x] ~Checked new UI names with UX convention~ No UI changes
- [x] ~Tested UI multi-edition + Undo/Redo~ No UI changes
- Other: 
Tested all manner of vector inputs in all modes of the Blend node to ensure that every operation was not outputting NaN and was expected output. 

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
None
**Halo Effect**: None, Low, Medium, High?
None
---
### Comments to reviewers
Notes for the reviewers you have assigned.
